build_tools/bazel/build.sh

```diff
@@ -37,9 +37,8 @@ echo "Running with test env args: ${test_env_args[@]}"
 # `bazel test //...` because the latter excludes targets tagged "manual". The
 # "manual" tag allows targets to be excluded from human wildcard builds, but we
 # want them built by CI unless they are excluded with "nokokoro".
-bazel query '//... except attr("tags", "nokokoro", //...)' | \
-  xargs bazel test ${test_env_args[@]} --define=iree_tensorflow=true \
-    --keep_going --test_output=errors
+bazel query '//iree/... + //test/... + //bindings/... except attr("tags", "nokokoro", //...)' | \
```

Comment:
Want to make any updates to the comment block above this like "Build and test everything" -> "Build and test supported paths"?

Reply:
Done